extensions/python/ExecutePythonProcessor.h

```diff
@@ -116,6 +116,14 @@ class ExecutePythonProcessor : public core::Processor {
     return description_;
   }
 
+  void setVersion(const std::string& version) {
+    version_ = version;
+  }
```

Comment:
Maybe we could filter empty values to not override the agent version

Reply:
Hm, I'm not sure. Is anyone going to set `ProcessorDetails.version = ""`? If they do, would they expect to see the MiNiFi version in the C2 server GUI?

Any comments/opinions from other people?